internal/database/repository/node.go

```diff
@@ -138,7 +137,6 @@ func (s *PostgresRepository) GetNodeUtilizations(
 	query := queryBuilder.Query()
 	args := queryBuilder.Args()
 	rows, err := s.dbpool.Query(ctx, query, args...)
-
 	if err != nil {
 		return nil, fmt.Errorf("could not get node utilizations from DB: %v", err)
```

Comment:
I'd suggest using `%w` so errors are wrapped and `errors.Is` works correctly, even though we may not have a concrete use case, this can often lead to silent bugs

Reply:
The intention is to modify the app, these are just formatting changes... I can apply the suggestion once we sync the nodes if you'd like. :relaxed: